### DISCUSSION TITLE
Set SEP on another user

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#2164] [Ideas]
In my product, all API calls are made via NodeJS connecting to an ODBC connection with one of two profiles. We don't have credentials for these profiles, so when we debug an API, we set up our SEP with whichever of the two users we want to debug.

Now that I have recently been able to try SEP support in Code4i, I realize that you cannot specify which user you want to 'catch' and debug their job. Can we add this please?

**Comment by sebjulliand** [comment]
I think this is in IBM's hands...

**Reply by worksofliam** [reply]
Shared with people at IBM who would the ones to do it.

**Comment by arco400** [comment]
I would really love this ability too. 
Have not had the chance to play with SEP via VScode yet, due to our system os version limitation, but in RDI we do a lot of web API debugging via SEP, where you for example can select the userprofile QTMHHTP1 to intercept.
This ability would be very welcome here.

**Reply by arco400** [reply]
Hi guys. Just curious. If I'm not mistaken, the host debug server is just the same for RDI and codefori. While debugging other users jobs is perfectly possible in RDI, wouldn't the same functions basicly be available for implementing in codefori?

Regards, Arco

**Reply by petterjohansson** [reply]
Also interested in this. 👍

**Reply by worksofliam** [reply]
Right now, the ability to debug on another user profile is not available.

**Reply by ewarwoowar-git** [reply]
Defo interested in this for the same reasons as arco400.

**Reply by heymchri** [reply]
This was made available in a recent update - just get the latest versions of the extensions.
![image](https://github.com/user-attachments/assets/b5f7e6d2-8856-4fe1-b23e-2828209baf99)
